arduino/f3f-arduino-timing/f3f-arduino-timing.ino

```diff
@@ -92,7 +97,7 @@ volatile debugStr debug = {0};
 volatile unsigned int i = 0;
 volatile byte temp = 0;
 volatile byte reset = true;
-
+volatile chronoMode chrono_mode = Training;
```

Comment:
Maybe it would be a better idea to set it in Race mode by default so that it is compatible with older version raspberry program ?
I admit that it should make no difference very quickly ...